src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +56,45 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return dateParams.length == 3
+                    ? Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]))
+                    : Optional.empty();
+        } catch (DateTimeException e) {
+            return Optional.empty();
```

Comment:
we can catch several particular exceptions in the same catch block, but in case of any trouble we should be able to identify them - for now you can use System.out.println(), later we will learn logging

move return Optional.empty(); to the end of method